python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -116,6 +116,13 @@ keys:
             forwarding_profile:
               type: str
               description: L2 protocol forwarding profile.
+        mac_timestamp:
+          type: str
+          valid_values: ["before-fcs", "replace-fcs", "header"]
+          description: |-
```

Comment:
please add in the description that the options are platform dependent

Reply:
Added.